tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker-test.php

```
@@ -512,6 +536,314 @@ function ( $pre, $args, $url ) {
+	 *
+	 * @return void
+	 */
+	public function test_sync_archives_skips_when_feature_disabled() {
+		$product = $this->create_simple_product_with_stock( 5 );
+		$this->sut->track_product_archive( $product->get_id() );
+
+		$this->sut->sync_archives();
+
+		$pending = get_option( WC_Stripe_Agentic_Commerce_Inventory_Tracker::PENDING_ARCHIVES_OPTION, [] );
+		$this->assertNotEmpty( $pending );
+	}
```

PHPMD (2.15.0)

[error] The method test_sync_archives_skips_when_feature_disabled is not named in camelCase. (undefined)

(CamelCaseMethodName)